package.json

```diff
@@ -5,8 +5,11 @@
   "license": "",
   "scripts": {
     "build": "remix build",
+    "predev": "yarn prepare",
```

Comment:
That way, the migrations automatically run when you launch `yarn dev`. Would that be an issue?

Reply:
Love it 😁